postman_tests/lib/datastore.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+import os
+import glob
+import json
+import csv
+import requests
+from . import common
+
+
+def get_api_url(config_path):
+    if os.path.exists(f"{config_path}/dev.json"):
+        config_file_path = f"{config_path}/dev.json"
+    else:
+        config_file_path = f"{config_path}/prod.json"
+    with open(config_file_path, 'r') as f:
+        data = json.load(f)
+    base_url = data["es_host"]
+    return f"http://{base_url}/entities/data/v1"
+
+
+def sync(es_data_path, config_path, mode):
+    """
+    Index data for every entity being tested into ElasticSearch
+
+    Parameters:
+    es_data_path (string): Path to the data/elastic_search directory
+
+    Returns:
+    None
+    """
+    for file_path in glob.glob(os.path.join(es_data_path, '*.csv')):
+        entity_name = common.get_entity_name(file_path)
+        print(f"Syncing {entity_name}, mode: {mode}")
+        contents = convert_csv_to_dict(file_path, mode)
+        url = get_api_url(config_path)
+        try:
+            req = requests.post(f"{url}/{entity_name}", data=json.dumps(contents))
+            req.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise e
```

Comment:
Redundant try/except. Less point in catching anything if we are just going to raise it without doing anything else